lib/ljglibs/gio/win32input_stream.moon

```diff
@@ -0,0 +1,22 @@
+-- Copyright 2014-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+require 'ljglibs.gio.input_stream'
+ffi = require 'ffi'
+core = require 'ljglibs.core'
+{:ref_ptr} = require 'ljglibs.gobject'
```

Comment:
We need to run this PR through the new linter later before merging btw to catch all of these things (e.g. unused var above).